src/test/java/org/embulk/input/marketo/delegate/LeadBulkExtractInputPluginTest.java

```diff
@@ -81,15 +81,15 @@ public void testRun() throws InterruptedException, IOException
         Mockito.verify(mockMarketoRestclient, Mockito.times(1)).waitLeadExportJobComplete(eq(exportId1), eq(task.getPollingIntervalSecond()), eq(task.getBulkJobTimeoutSecond()));
         Mockito.verify(mockMarketoRestclient, Mockito.times(1)).startLeadBulkExtract(eq(exportId2));
         Mockito.verify(mockMarketoRestclient, Mockito.times(1)).waitLeadExportJobComplete(eq(exportId2), eq(task.getPollingIntervalSecond()), eq(task.getBulkJobTimeoutSecond()));
-        String filterField = "updatedAt";
```

Comment:
These are pretty naive change of mine without full context, help me take a look @taikhuu